src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxRewriter/BodyBlockCSharpSyntaxRewriter.cs

```diff
@@ -48,7 +48,9 @@ public BodyBlockCSharpSyntaxRewriter(string? exceptionMessage)
             }
 
             string returnType = rs.ReturnType.ToString();
-            if (returnType != "void" && returnType != "System.Void")
+            if (returnType != "void" && returnType != "System.Void" ||
```

Comment:
This is easy to get/read wrong. Please add braces. I assume it should be `((returnType != "void" && returnType != "System.Void") || X)`

Reply:
Agree. Thanks